www/templates/control/main-control.html

```diff
@@ -24,151 +38,200 @@
           style="position: absolute; height: 100%; inset: 0; display: flex; opacity: 0;">
       </div>
     </div>
-    <div class="control-list-item">
+    <!-- <setting-row text-key="'control.reminders-time-of-day' | i18next: {time: settings.notification.prefReminderTime}" icon-name="'clock'" action = "">
+      <input type="time" name="timeOfDay" value="{{settings.notification.prefReminderTimeOnLoad}}"
```

Comment:
I would also leave this one how it was, for now

We need to find a library for time/date pickers
I've been eyeing this one: [react-native-paper-dates](https://github.com/web-ridge/react-native-paper-dates) but haven't implemented it anywhere yet.

For the date picker on the label screen I just left it as HTML and wrapped it in a React component